sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -530,11 +552,24 @@ export interface ListOperationsOptions extends OperationOptions {
 }
 
 // @public
-export interface OperationDetails extends OperationSummary {
+export interface OperationDetails {
+    apiVersion?: string;
+    createdDateTime: Date;
     error?: ErrorModel;
-    result?: Record<string, unknown>;
+    kind: "documentModelBuild" | "documentModelCompose" | "documentModelCopyTo";
+    lastUpdatedDateTime: Date;
+    operationId: string;
+    percentCompleted?: number;
+    resourceLocation: string;
+    status: OperationStatus;
+    tags?: {
+        [propertyName: string]: string;
+    };
 }
 
+// @public (undocumented)
+export type OperationDetailsUnion = OperationDetails | DocumentModelBuildOperationDetails | DocumentModelComposeOperationDetails | DocumentModelCopyToOperationDetails;
+
```

Comment:
This generated type is pretty weird. All of the variants `extend OperationDetails`.